Client/mods/deathmatch/logic/CClientColModel.h

```diff
@@ -46,7 +99,8 @@ class CClientColModel : public CClientEntity
     void InternalRestore(unsigned short usModel);
 
     class CClientColModelManager* m_pColModelManager;
-
-    CColModel*                m_pColModel;
-    std::list<unsigned short> m_Replaced;
+    unsigned short                m_usVerticesCount;
+    CColModel*                    m_pColModel;
+    std::list<unsigned short>     m_Replaced;
+    bool                          b_hasChanged; // For updating bounding box, update only if necessary. True default due object loaded collision could have wrong bounding box.
```

Comment:
Preferably just put this own comment on its own line separate from the rest. It's too long!
```suggestion

    // m_bHasChanged should be set to true when the bounding box has changed.
    // This ensures that the bounding box is only recalculated when necessary.
    // This is true by default as an object loaded collision could have the wrong bounding box.
    bool b_hasChanged = true;
```

Also, the variable should probably be `m_bHasChanged`